src/main/module/module-info.java

```diff
@@ -6,8 +6,13 @@
 
 	exports org.junitpioneer.vintage;
 	exports org.junitpioneer.jupiter;
+	exports org.junitpioneer.jupiter.cartesian;
 	exports org.junitpioneer.jupiter.params;
 
+	opens org.junitpioneer.jupiter to org.junit.platform.commons;
+	opens org.junitpioneer.jupiter.cartesian to org.junit.platform.commons;
+	opens org.junitpioneer.jupiter.params to org.junit.platform.commons;
```

Comment:
Don't we have to open `vintage` as well?

Reply:
Good point! Since `vintage` also operates on top of the Jupiter API, I will add it, too.